content/pt-br/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Namespaces
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+Em Kubernetes, _namespaces_ disponibilizam um mecanismo para isolar grupos de recursos dentro de um único cluster. Nomes de recursos precisam ser únicos dentro de um namespace, porém podem se repetir em diferentes namespaces. Escopos baseados em namespaces são aplicáveis apenas para objetos com namespace _(como: Deployments, Services, etc)_ e não em objetos que abrangem todo o cluster _(como: StorageClass, Nodes, PersistentVolumes, etc)_.
```

Comment:
I was thinking, here we can use `No Kubernetes` instead of `Em Kubernetes`, what you think?